manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -117,6 +117,31 @@ spec:
         severity: warning
   - name: openshift-etcd.rules
     rules:
+    - alert: etcdHighNumberOfFailedGRPCRequests
+      annotations:
+        description: 'etcd cluster "{{ $labels.job }}": {{ $value }}% of requests for {{ $labels.grpc_method }} failed on etcd instance {{ $labels.instance }}.'
```

Comment:
Should this one also have a runbook?

Reply:
No it's warning and only critical alerts, for now, have runbooks as per the monitoring team and SRE folks reqs.